frontend/src/Explorer/View/Layout.purs

```diff
@@ -28,7 +28,7 @@ view state =
                     [ P.className "explorer-content" ]
                     [ case state.route of
                           Dashboard -> dashboardView state
-                          (Transaction hash) -> transactionView state hash
+                          (Tx id) -> transactionView state
```

Comment:
Both class names (`explorer-content`, `transaction-header`) are used only once. There are just few (not much) classes, which are shared with different view. Most of them are defined in `global.css`. Anyway, I understand your concerns, but would recommend to do this job if we have more time.